modules/cpmso-yaml-failure-domain-openstack.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-configuration.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-failure-domain-openstack_{context}"]
+= Sample {rh-openstack} failure domain configuration
+// TODO: Replace that link.
+The control plane machine set concept of a failure domain is analogous to existing {rh-openstack-first} concept of an link:https://docs.openstack.org/cinder/latest/admin/availability-zone-type.html[_Availability Zone (AZ)_]. The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible. 
+
+When configuring {rh-openstack} failure domains in the control plane machine set, you must specify the availability zone name and the subnet to use. 
```

Comment:
the subnet?

Reply:
Not a fan of this whole description. Will replace.